docs/Query-Library-Transfer.md

```diff
@@ -100,8 +100,7 @@ $ cat appState.json | jq '.data.globalState.queries'
 # Extracting/Replacing the Query Library
 
 For ease of use, we've created a simple Python script
-[`qlib_util.py`](https://raw.githubusercontent.com/brimdata/brim/master/scripts/util/qlib_util.py)
-that can easily perform this.
+[`qlib_util.py`](../scripts/util/qlib_util.py) that can easily perform this.
```

Comment:
```suggestion
[`qlib_util.py`](https://raw.githubusercontent.com/brimdata/brim/main/scripts/util/qlib_util.py) that can easily perform this.
```

The original change was going to break the [wiki](https://github.com/brimdata/brim/wiki). When I set up the wiki, there was a strong desire in the team to still have the original Markdown docs live in the main code repo, so we have a [docs-publish.yml](https://github.com/brimdata/brim/blob/main/.github/workflows/docs-publish.yml) Actions Workflow that copies those Markdown files from `docs/` into the appropriate spots in the separate meta-repo that holds the actual wiki. Links in that wiki repo are not relative to the main repo, so I'm pretty sure we need to maintain the original absolute link with `main` in it. Sorry if that's disappointing.

Reply:
Thanks for pointing this out!

25388b4 restores absolute under `docs/`.